flybywiresim/a32nx
# feat: Button press sound for Remote MCDU clicks

Closes #6632 

## Summary of Changes
Adds a button press sound to the Remote MCDU.
Sound can be turned on by adding "/sound" to the URL. 

## Screenshots (if necessary)
https://user-images.githubusercontent.com/16833201/153566539-32d1d0d2-6f85-4fb8-8cc1-e562c402d5c8.mp4


Discord username (if different from GitHub): Cdr_Maverick#6475

## Testing instructions
- Check that when Remote MCDU is loaded with http://localhost:8125/sound you can hear a button press sound with each click on a button.
- Check that there is no sound when not adding "/sound" to the URL. 
- Check that other parameters like "/fullscreen" still work
- General check if nothing else has been broken when using the Remote MCDU


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
